lib/tandem/tandemSimulator.js

```diff
@@ -85,6 +90,16 @@ exports.make = function(config){
           events.push(currBasal);
         }
       }
+
+      if (event.deliveryType === 'suspend') {
+        setCurrStatus('suspended');
+      }
+      else if (event.deliveryType === 'resume') {
```

Comment:
Oh, I missed this before. There's no such thing as a `resume` `deliveryType`. We should just use a distinction between `suspended` and `null` for now, I guess.